swarm/pss/prox_test.go

```diff
@@ -92,7 +93,7 @@ func (d *testData) setDone() {
 	d.handlerDone = true
 }
 
-func getCmdParams(t *testing.T) (int, int) {
+func getCmdParams(t *testing.T) (int, int, time.Duration) {
```

Comment:
- Why do we want to get the timeout from command line?
- Why do we use positional arguments instead of flags? The latter provide names to parameters - easier to read - and harder to mess up by wrong ordering.

Reply:
@frncmx 

* cos more nodes and more messages may need longer timeout
* not sure what you mean by flags? If you mean as in executable flags - the test is not supposed to work that way. It's only a way of defining different configurations to run the test with. It's a pattern used other places in the swarm code.

Reply:
As I later realized that we are not parsing command line params (~~`Cmd`~~) my comment is irrelevant. If my current understanding is correct the naming is wrong.

Also I suggested later table driven tests instead.